File: Algorithms/BubbleSort/BubbleSort.cpp
#include <bits/stdc++.h> 
using namespace std; 
  
  
void bubbleSort(int arr[], int n)  
{  
    for i in range(n): 
        for j in range(0, n-i-1): 
            if arr[j] > arr[j+1] : 
                arr[j], arr[j+1] = arr[j+1], arr[j]
}  
  
/* Function to print an array */
void printArray(int arr[], int size)  
{  
    int i;  
    for (i = 0; i < size; i++)  
        cout << arr[i] << " ";  
    cout << endl;  
}  
  
int main()  
{  
    int arr[] = {100, 75, 25, 150, 10, 45, 32, 99};  
    int n = sizeof(arr)/sizeof(arr[0]);  
    bubbleSort(arr, n);  
    cout<<"Sorted array: \n";  
    printArray(arr, n);  
    return 0;  
}  

File: Algorithms/BubbleSort/BubbleSort.js

function bubble_Sort(arr){

    let len = arr.length;
        for (let i = 0; i < len; i++) {
            for (let j = 0; j < len; j++) {
                if (arr[j] > arr[j + 1]) {
                    let tmp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = tmp;
                }
            }
        }
    return arr;
}
console.log(bubble_Sort([3, 0, 2, 5, -1, 4, 1]));
